<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DictEditorWindow</class>
 <widget class="QDialog" name="DictEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>628</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>628</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dictionary Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>391</height>
    </rect>
   </property>
   <widget class="QListWidget" name="wordsList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>191</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="wordsLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Слова</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Новое слово</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prefixEdit_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="declSelect_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Именительный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Родительный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дательный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Винительный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Творительный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Предложный</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="prefixEdit_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="declSelect_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="prefixEdit_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="declSelect_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="prefixEdit_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="declSelect_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="prefixEdit_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="declSelect_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="prefixEdit_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="declSelect_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>280</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
